frontend/package.json

```diff
@@ -130,5 +130,8 @@
     "ignorePatterns": [
       "cypress"
     ]
+  },
+  "dependencies": {
```

Comment:
@NicolaeUnleash @nunogois Are there no frontend dependencies? Or to be moved to `devDependencies`...?

Reply:
I believe this belongs in `devDependencies` instead yeah.

Reply:
Updated. What's the motivation behind having it in `devDependencies`? Will it not cause a lot of unnecessary modules to also come in for a production build?

Reply:
That's a good question and I can't give you an answer off the top of my head on why we decided to go with this approach, but I don't think it matters much with Vite anyways: https://github.com/vitejs/vite/discussions/1803